src/data/index.ts

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
what is this? 
1- It should be the part of helper.
2- Why we only have 1 error code & message?

Reply:
this is the default message. Otherwise, the message and both codes are provided when the error is thrown.